sale_planner_calendar/models/res_users.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2024 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import json
+from datetime import datetime
+
+import pytz
+
+from odoo import _, api, fields, models, modules
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    def _get_sale_planner_calendar_events_domain(self):
+        return [
+            ("user_id", "=", self.env.user.id),
+            ("target_partner_id", "!=", False),
+            ("sale_planner_state", "=", "pending"),
+        ]
+
+    @api.model
+    def get_action_sale_planner_calendar_event(self):
+        return self.env["ir.actions.act_window"]._for_xml_id(
```

Comment:
Maybe would be interest return by default de kanban view...

Reply:
Kanban is now the default view for this action